bench/preproccont.fpcore

```diff
@@ -0,0 +1,19 @@
+; -*- mode: scheme -*-
```

Comment:
Is this file just duplicates of existing benchmarks? Because if so we should not commit it.

Reply:
yeah sorry forgot to delete before PR